stories/text/text.stories.js

```diff
@@ -0,0 +1,167 @@
+export default {
+    title: 'Components/Text',
+    parameters: {
+        description: `The text component displays text inside a form, table, or any other content area.
+        It is generally used throughout the entire application and is responsive to all screen sizes.`,
+        tags: ['f3', 'a11y', 'theme'],
+        components: ['text']
+    }
+};
+
+export const defaultExample = () => `<p class="fd-text">
+    Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt
+    ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut
+    aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore
+    eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia
+    deserunt mollit anim id est laborum.</p>
+`;
+
+defaultExample.storyName = 'Default';
+defaultExample.parameters = {
+    docs: {
+        iframeHeight: 100,
+        storyDescription: `The default text component can display lines of text that wrap to the next line
+        once they reach the end of the content container width.`
+    }
+};
+
+export const whitespace = () => `
+    <h3>No indents and whitespace</h3>
```

Comment:
Hey Alexei, last thing I promise! Because it's a negative sentence it's "No indents or whitespace"

Reply:
It's ok, no worries :)

Reply:
Updated PR